### DISCUSSION TITLE
Lightning Address

### DISCUSSION THREAD

**Discussion by riley-parks** [btcpayserver/btcpayserver#4993] [Ideas / Feature Requests]
Remember seeing it somewhere on the dashboard before, cant find it now.

Was it removed for a reason, if so, what for? And is it possible to add it back for my installation?

**Comment by dstrukt** [comment]
Was not removed - I forget if it's installed by default, but if not, it can be found and installed as a Plugin!

Hope this helps.

**Comment by riley-parks** [comment]
Tried to search for it again, and in the manage plugin menu, but no luck.

Could it be I am running it on my local network, and without TOR?

Riley

**Reply by dennisreimann** [reply]
LNURL has to be enabled for the store, otherwise it won't show up.

**Comment by High1741** [comment]
[](hugh1741) bitcoin:bc1q7x89emrmakypa38508g2lc7suyjqfarecyq4n8?amount=0.00287071

**Comment by High1741** [comment]
Need this really bad

**Comment by mp21-org** [comment]
https://blog.btcpayserver.org/bitcoin-bonus-scam/

Am 10. Juni 2023 10:58:29 MESZ schrieb High1741 ***@***.***>:
